### DISCUSSION TITLE
How to display date in mm.dd.yyyy instead of dd.mm.yyyy format?

### DISCUSSION THREAD

**Discussion by prudhvi-git** [taiga-family/taiga-ui#611] [Q&A] [answered]
For DateTime fields (InputDate, InputDateRange,..), I don't see any format option to display selected date in mm.dd.yyyy format. 
Looking at the code, TuiDay has formattedDay hardcoded to dd.mm.yyyy format. 

Is there a way to format selected date on Date Input elements?

![image](https://user-images.githubusercontent.com/7905532/128570512-9b5d311c-1865-4754-9d9b-62f8a0bca5c0.png)

**Comment by waterplea** [comment, marked as answer]
This is now possible with `TUI_DATE_FORMAT` token. See **Date localization** example:
https://taiga-ui.dev/components/input-date#date-localization
